### PR TITLE
Meta tags are in partial

### LINKED ISSUE #166: Refactor meta tags into partial
In `app/views/layouts/application.html.haml` we have [some meta tags](https://github.com/hackedu/website/blob/master/app/views/layouts/application.html.haml#L7-L11) that should be refactored out into their own partial.

### PR DESCRIPTION
**Pull request by maxwofford**
@zachlatta Would you please review this?
This closes #166

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -4,11 +4,7 @@
     %meta{charset: 'utf-8'}
     %meta{content: 'width=device-width, initial-scale=1.0', name: 'viewport'}
```
Comment (zachlatta) [inline app/views/layouts/application.html.haml]: Can you remove this extra line?

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -4,11 +4,7 @@
...
 
-    %meta{name: 'title', content: 'hackEDU'}
-    %meta{name: 'description', content: 'Be a part of the global movement of free student-led coding clubs.'}
-    %meta{name: 'og:type', content: "website"}
-    %meta{name: 'og:url', content: "https://hackedu.us"}
-    %meta{name: 'og:image', content: "#{ image_path('group_photo_no_gradient.jpg') }"}
+    = render 'layouts/meta'
```
Comment (zachlatta) [inline app/views/layouts/application.html.haml]: This as well?

**Comment by zachlatta** [comment]
Can you also reword your commit to be imperative?

> Move social meta tags to partial

File: app/views/layouts/_meta.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+%meta{name: 'title', content: 'hackEDU'}
+%meta{name: 'description', content: 'Be a part of the global movement of free student-led coding clubs.'}
+%meta{name: 'og:type', content: "website"}
+%meta{name: 'og:url', content: "https://hackedu.us"}
+%meta{name: 'og:image', content: "#{ image_path('group_photo_no_gradient.jpg') }"}
```
Comment (zachlatta) [inline app/views/layouts/_meta.html]: Perhaps `_social_meta.html.haml` would be a better name for this? We have other meta tags that are in `application.html.haml` that shouldn't be in this file (like the viewport declaration).

Also, should the extension of this file end in `.haml`?

**Comment by MaxWofford** [comment]
@zachlatta Comments addressed :)

**Pull request merged**